dts/bindings/gpio/microchip,mcp23x17-spi.yaml

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2020 Geanix ApS
+# Copyright (c) 2020 Prevas A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: >
```

Comment:
Please use `|` instead of `>`. See https://github.com/nrfconnect/sdk-nrf/commit/b9240a3cbc2012351a758bdf19313b80fcb3821e for rationale.

Reply:
Wrong link, sorry. I meant b9240a3cbc2012351a758bdf19313b80fcb3821e.